src/core/maps/area_occupancy_estimator.h

```diff
@@ -6,32 +6,44 @@
 #include <type_traits>
 
 #include "cell_occupancy_estimator.h"
-
+/*!
+ * \brief Derived class from CellOccupancyEstimator to calculate the value of occupancy of one cell
```

Comment:
Better: "A strategy that estimates grid cell's occupancy based on how a laser beam passes the cell."